src/antlr4/Cypher.g4

```diff
@@ -185,8 +184,12 @@ oC_UpdatingClause
 oC_ReadingClause
     : oC_Match
         | oC_Unwind
+        | oC_CallTableFunc
         ;
 
+oC_CallTableFunc
```

Comment:
oC_InQueryCall

Let's use the Neo4j naming here and propagate the naming change to other classes and files